kythe/go/extractors/config/runextractor/mavencmd/pom_xml_modifier.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mavencmd
+
+import ()
```

Comment:
delete this?

Reply:
Yea removed from this change in '943f77f

Can submit that later today

Honestly including it here was accidental anyways.